apps/fyle/tasks.py

```
@@ -829,3 +829,141 @@ def delete_expenses_in_db(expense_ids: list[int], workspace_id: int) -> None:
+        worker_logger.info("Deleting expense group %s for report %s", expense_group.id, report_id)
+
+        with transaction.atomic():
+            delete_expense_group_and_related_data(expense_group, workspace.id)
+
+        deleted_count += 1
+
+    worker_logger.info("Completed deletion for report %s in workspace %s - deleted: %s, skipped: %s",
+                report_id, workspace.id, deleted_count, skipped_count)
+
+
+def _handle_expense_ejected_from_report(expense: Expense, expense_data: dict, workspace: Workspace) -> None:
```

Ruff (0.14.0)

Unused function argument: `expense_data`

(ARG001)